packages/plugin/API.md

```diff
@@ -519,6 +519,27 @@ function provideDefinitionHandler(document: theia.TextDocument, position: theia.
 The handler will be invoked each time when a user executes `Go To Definition` command.
 It is possible to return a few sources, but for most cases only one is enough. Return `undefined` to provide nothing.
 
+#### Reference provider
+
+It is possible to provide reference sources for a symbol from within plugin.
+To do this one should register corresponding provider. For example:
+
+```typescript
+const documentsSelector: theia.DocumentSelector = { scheme: 'file', language: 'typescript' };
+const handler: theia.ReferenceProvider = { provideReferences: provideReferencesHandler };
+
+const disposable = theia.languages.registerReferenceProvider(documentsSelector, handler);
+
+...
+
+function provideRefereencesHandler(document: theia.TextDocument, position: theia.Position, context: theia.ReferenceContext): theia.ProviderResult<theia.Location[]> {
```

Comment:
it seems that there is a typo on provideRefereencesHandler (two e)

Reply:
Yes, there is. I'll provide a fixup